editor/scene_tree_dock.cpp

```diff
@@ -2628,9 +2624,6 @@ void SceneTreeDock::attach_script_to_selected(bool p_extend) {
 		}
 	}
 
-	script_create_dialog->connect("script_created", callable_mp(this, &SceneTreeDock::_script_created));
-	script_create_dialog->connect("popup_hide", callable_mp(this, &SceneTreeDock::_script_creation_closed), varray(), CONNECT_ONESHOT);
-	script_create_dialog->set_inheritance_base_type("Node");
```

Comment:
Are you sure that this one is unnecessary? I assume it was put here to reset things which might have been modified in the Create New Script variant of this dialog.

Reply:
This only thing it did was to disconnect the "script_created" signal. I believe this was added originally only to avoid the "signal already connected" message, because I see no other reason to disconnect this signal otherwise.

Since now the signal connection is done once and not every time the attach_script_to_selected function is ran, I see no reason to keep this.

Reply:
I meant the `set_inheritance_base_type` specifically.